### PR TITLE
Add an initial port of a subset of Qt6

### PR DESCRIPTION
**Pull request by Dennisbonke**
This PR adds the following ports:

- `double-conversion`;
- `brotli`;
- `pcre2`;
- `qtbase`;
- `qtsvg`;
- `qtlanguageserver`;
- `qtshadertools`;
- `qtimageformats`;
- `qtdeclarative`;
- `qtmultimedia`;
- `libremines`.

Blocked on managarm/mlibc#406
Blocked on #176

### CODE REVIEW

File: bootstrap.d/dev-qt.yml [hunk 1]
```diff
@@ -0,0 +1,395 @@
...
+      git: 'https://code.qt.io/qt/qtbase.git'
+      tag: 'v6.3.0'
+      version: '6.3.0'
+    tools_required:
+      - system-gcc
+      - host-cmake
+      - host-qt6
+      - host-pkg-config
+      - virtual: pkgconfig-for-target
+        triple: x86_64-managarm
```
Comment (qookei) [inline bootstrap.d/dev-qt.yml:51]: This should be `triple: "@OPTION:arch-triple@"`

**Review by qookei — CHANGES_REQUESTED** [review]
Looks good apart from the comment

File: bootstrap.d/dev-qt.yml (hunk 1 above)
Comment (Dennisbonke) [reply bootstrap.d/dev-qt.yml:51]: Good catch, fixing.

**Review by qookei — APPROVED** [review]
LGTM

**Pull request merged**